Documentation/Updating.md

```diff
@@ -101,6 +101,11 @@ This version of MRTK modifies the steps required for eye gaze setup. The _'IsEye
 
 For more information on these changes and complete instructions for eye tracking setup, please see the [eye tracking](EyeTracking/EyeTracking_BasicSetup.md) article.
 
+### Eye gaze pointer behavior in 2.4.0
+
+The eye gaze default pointer behavior have been modified to match the head gaze default pointer behavior. An eye gaze pointer will automatically be suppressed once a hand is detected. 
+Details about gaze and hand setups can be found in the [eyes and hands](EyeTracking/EyeTracking_EyesAndHands.md) article.
```

Comment:
Nit: If you think it would be helpful, having a direct link to the section (#how-to-keep-gaze-pointer-always-on) would bring folks directly back to the previous behavior that they had.

If the idea is to get folks to read the entire thing though then it's fine as-is.